app/views/admin/petitions/edit.html.erb

```diff
@@ -2,7 +2,11 @@
 
   <%= render :partial => 'petition_details', :locals => { :f => f } %>
 
-  <div>
+  <div class="row">
+    <%= link_to 'Edit petition content', admin_petition_petition_details_path(@petition) %>
+  </div>
```

Comment:
Oh no, actually I can't do it - as the editing content is only for petitions that are in moderation so it doesn't include petitions that are already open/rejected/etc